frontend/src/components/project/ProjectOverview.js

```diff
@@ -49,6 +49,9 @@ const useStyles = makeStyles((theme) => ({
     display: "flex",
     marginTop: theme.spacing(3),
     justifyContent: "space-between"
+  },
+  smallScreenHeader {
+    fontSize: "calc(1.6rem + 6 * ((100vw - 320px) / 680))"
```

Comment:
You'll need a ":" for this to work:
```suggestion
  smallScreenHeader: {
    fontSize: "calc(1.6rem + 6 * ((100vw - 320px) / 680))"
```